hack/azure-cluster-up.sh

```diff
@@ -63,8 +72,12 @@ echo -e "\nSpinning up Azure Kubernetes Cluster $CLUSTER_NAME"
 az aks create --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --enable-addons monitoring --generate-ssh-keys --service-principal $username --client-secret $password
 
 echo -e "\nGetting Credentials for Cluster $CLUSTER_NAME"
-az aks get-credentials --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME
+az aks get-credentials --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --overwrite-existing
 echo -e "\n"
 
-DRIVER_NAME=$(dirname "$(realpath ${BASH_SOURCE[0]})")
-source "$DRIVER_NAME/cosi-install.sh"
\ No newline at end of file
+if [ $VERSION = "local" ] || [ $VERSION = "push" ];
+then
+    DRIVER_NAME=$(dirname "$(realpath ${BASH_SOURCE[0]})")
+    # shellcheck source=./hack/cosi-install.sh
```

Comment:
remove comment

Reply:
The comment is necessary to prevent shellcheck errors. Because I'm not using a direct path, shellcheck can't really follow the dynamic source, and needs to be told path to use. Alternatively, I could just use a direct path, I just designed it so it wouldn't matter where you were running the script from.

Reply:
Got it